### DISCUSSION TITLE
sv_84 and tv_84 difference

### DISCUSSION THREAD

**Discussion by mittermichal** [etlegacy/etlegacy#2862] [Q&A] [answered]
@ryzyk-krzysiek could you please just shortly summarize how these 2 differ in terms of what they store?
I wonder if just archiving and distributing only tv_84 is enough. sv_84 can be on the other hand nice alternative if one doesn't want to bother with ETLTV slave setup, but then you in your UDT PR https://github.com/mightycow/uberdemotools/pull/2 only implemented tv_84 support right? Or are they very similar and adding support for them in UDT is not much work?

I dunno what your vision with all of this is if you have any, you make this :goat:  features and I can't keep up with what is it supposed to be used for :smile:

**Comment by mittermichal** [comment]
partial/full answer basically: https://github.com/etlegacy/etlegacy/issues/2861#issuecomment-2402874606

**Comment by ryzyk-krzysiek** [comment, marked as answer]
sv_84 differs quite a bit, the whole "message" is constructed differently and doesn't follow the quake protocol at all beside playerstate/entitystate having same structure ofc, it also saves bigger `sharedEntity` structure compared to tv demos. It just doesn't make sense adding it to UDT at all in my opinion.

sv_84 has nothing special that is useful compared to tv_84.
